index.html

```diff
@@ -10469,16 +10469,30 @@ <h3>application/ld+json</h3>
-            To request or specify <a href="#expanded-document-form">expanded JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#expanded</code> SHOULD be used.
-            To request or specify <a href="#compacted-document-form">compacted JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#compacted</code> SHOULD be used.
-            To request or specify <a href="#flattened-document-form">flattened JSON-LD document form</a>,
-            the URI <code>http://www.w3.org/ns/json-ld#flattened</code> SHOULD be used.
-            Please note that, according [[HTTP11]], the value of the <code>profile</code>
-            parameter has to be enclosed in quotes (<code>"</code>) because it contains
-            special characters and, if multiple profiles are combined, whitespace.</p>
+          <p>This specification defines four values for the <code>profile</code> parameter.</p>
+          <dl>
+            <dt><code>http://www.w3.org/ns/json-ld#expanded</code></dt>
+            <dd>To request or specify <a href="#expanded-document-form">expanded JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#compacted</code></dt>
+            <dd>To request or specify <a href="#compacted-document-form">compacted JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#flattened</code></dt>
+            <dd>To request or specify <a href="#flattened-document-form">flattened JSON-LD document form</a>.</dd>
+            <dt><code>http://www.w3.org/ns/json-ld#framed</code></dt>
+            <dd>To request or specify <a href="#framed-document-form">framed JSON-LD document form</a>.</dd>
+          </dl>
+          <p>
+            When used as a <a data-cite="RFC2048#section-2.2.3">media type paramter</a> [[RFC2048]]
+            in an <a data-cite="rfc7231#section-5.3.2">HTTP Accept header</a> [[RFC7231]],
+            the value of the <code>profile</code> parameter MUST be enclosed in quotes (<code>"</code>) if it contains
+            special characters such as whitespace, when multiple profiles are combined.</p>
+          <p>In addition to the <code>profile</code> values described above,
+            the <code>profile</code> parameter MAY include one additional URL, which is interpreted as
+            the location of either a <a>context</a> or <a>frame</a>, depending on the other profile
+            parameters included. When used without an defined profile URI, such a URL
+            is interpreted as the location of a <a>context</a>.</p>
+          <p>JSON-LD processors MAY place restrictions on supported profiles,
+            MAY provide defaults for <a>context</a> and <a>frame</a> URLs,
+            and MAY restrict client-provided URLs for <a>contexts</a> and <a>frames</a>.</p>
           <p>When processing the "profile" media type parameter, it is important to
```

Comment:
@iherman this is why the "limit profile parameter use to URI’s" of our resolution text is important, and why it suggests creating a new issue for that URL dereferencing approach--which Gregg's has fleshed out a bit in the API PR https://github.com/w3c/json-ld-api/pull/56 (uses `Link` headers instead of the `profile=` media type parameter).